.github/workflows/ci-build.yml

```diff
@@ -0,0 +1,29 @@
+name: CI
+
+on:
+  pull_request:
+
+   
+jobs:
+  Code-Quality-Checks:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix: 
+        scripts:
+          - run-linter
+          - check-docstyle
+          - detect-dead-code
+          - detect-common-errors
+          - measure-cyclomatic-complexity
+          - measure-maintainability-index    
+    steps:
+    - uses: actions/checkout@v2
+    - name: Set up Python 3.6
+      uses: actions/setup-python@v2
+      with:
+        python-version: '3.6'
+    - name: Install virtualenv
+      run: pip install -U pip && pip install virtualenv
+    - name: ${{ matrix.scripts }}
+      run: bash qa/${{ matrix.scripts }}.sh;
+      
```

Comment:
Can you add another job to build docker image, just to sanity check Dockerfile.

Reply:
okay